crypto/fipsmodule/CMakeLists.txt

```diff
@@ -149,6 +150,7 @@ if(PERL_EXECUTABLE)
   perlasm(p256-armv8-asm.${ASM_EXT} ec/asm/p256-armv8-asm.pl)
   perlasm(p256_beeu-armv8-asm.${ASM_EXT} ec/asm/p256_beeu-armv8-asm.pl)
   perlasm(rdrand-x86_64.${ASM_EXT} rand/asm/rdrand-x86_64.pl)
+  perlasm(rndr-armv8.${ASM_EXT} rand/asm/rndr-armv8.pl)
```

Comment:
Why not use intrinsics?  We could use int __rndr (uint64_t *) as defined here (https://developer.arm.com/documentation/101028/0012/8--Data-processing-intrinsics)

Reply:
intrinsic support is compiler dependent, it's possible that it's not fully portable